config/env/development.js

```diff
@@ -65,5 +65,6 @@ module.exports = {
       }
     }
   },
-  livereload: true
+  livereload: true,
+  seedDB: true
```

Comment:
Are you sure?  Almost every other framework I have worked with auto seeds on development...